tasks/grunt-release.js

```diff
@@ -10,8 +10,16 @@ var shell = require('shelljs');
 var semver = require('semver');
 
 module.exports = function(grunt){
+  grunt.registerTask('bump-only', 'bump version only', function(type){
```

Comment:
Please also change `bump-only` to `release-bump` since `bump` is a subset of `release` and I believe we should "namespace" it here.